### PR TITLE
Add documentation to aid understanding in challenge

### PR DESCRIPTION
**Pull request by ashwinsrnth**
It's not entirely obvious what `running` does in the challenge on 'Testing Assertions'. Added some documentation to make it a little bit more clear.

### CODE REVIEW

File: 06-func.html [hunk 1]
```diff
@@ -376,7 +376,7 @@ <h2 id="defining-defaults">Defining Defaults</h2>
 </div>
 <div id="rescaling-an-array" class="challenge">
 <h2>Rescaling an array</h2>
-<p>Write a function <code>rescale</code> that takes an array as input and returns a corresponding array of values scaled to lie in the range 0.0 to 1.0. (If <span class="math"><em>L</em></span> and <span class="math"><em>H</em></span> are the lowest and highest values in the original array, then the replacement for a value <span class="math"><em>v</em></span> should be <span class="math">(<em>v</em> − <em>L</em>)/(<em>H</em> − <em>L</em>)</span>.) Be sure to give the function a docstring.</p>
+<p>Write a function <code>rescale</code> that takes an array as input and returns a corresponding array of values scaled to lie in the range 0.0 to 1.0. (If <span class="math"><em>L</em></span> and <span class="math"><em>H</em></span> are the lowest and highest values in the original array, then the replacement for a value <span class="math"><em>v</em></span> should be <span class="math">(<em>v</em> − <em>L</em>) / (<em>H</em> − <em>L</em>)</span>.) Be sure to give the function a docstring.</p>
```
Comment (gvwilson) [inline 06-func.html]: This seems like an accidental change (just puts spaces around a division slash).

File: 07-defensive.md [hunk 2]
```diff
@@ -648,9 +648,18 @@ not more.
...
 > and for each one,
 > give an example of input that will make that assertion fail.
->  
+> 
 > ~~~ {.python}
 > def running(values):
+>     '''Given a sequence of values, return a list containing
+>        the running totals at each index.
+>
+>        >>> running([1, 2, 3, 4])
```
Comment (gvwilson) [inline 07-defensive.md]: Learners won't have seen doctests, so this will confuse them more than enlighten.

**Comment by ashwinsrnth** [comment]
Thanks, @gvwilson. I've put the documentation in a more familiar format to learners. How does this look?

**Comment by gvwilson** [comment]
Looks good to me.

**Comment by kynan** [comment]
Apart from the incidental change pointed out by @gvwilson this looks good to me. @tbekolay?

**Comment by tbekolay** [comment]
LGTM, thanks @ashwinsrnth!

**Pull request merged**